src/main/java/com/softserve/teachua/controller/CenterController.java

```diff
@@ -87,6 +87,11 @@ public Page<ClubResponse> getCenterClubsByCenterId(@PathVariable Long id,
         return centerService.getCenterClubsByCenterId(id, pageable);
     }
 
+    @GetMapping("centers/allclubs/{id}")
+    public Object getAllCenterClubsByCenterId(@PathVariable Long id) {
```

Comment:
I'd also highly recommend following the project's API routes convention. If you have chosen to associate this functionality with the `CenterController` and there is a similar route, consider using the parameter, like
 ```suggestion
     @GetMapping("/centers/clubs", params = "centerId")
     public Object getAllCenterClubsByCenterId(@RequestParam("centerId) Long id) {
```


I'd relate such functionality to `ClubController`, though.